Tiltfile

```diff
@@ -371,7 +371,7 @@ def deploy_worker_templates(template, substitutions):
     else:
         calico_values = "./templates/addons/calico/values.yaml"
     flavor_cmd += "; " + helm_cmd + " repo add projectcalico https://docs.tigera.io/calico/charts; " + helm_cmd + " --kubeconfig ./${CLUSTER_NAME}.kubeconfig install calico projectcalico/tigera-operator -f " + calico_values + " --namespace tigera-operator --create-namespace"
-    if "intree-cloud-provider" not in flavor_name and "ipv6" not in flavor_name:  # TODO: remove ipv6 once https://github.com/kubernetes-sigs/cloud-provider-azure/issues/3401 is fixed.
+    if "intree-cloud-provider" not in flavor_name:
```

Comment:
Does this fix #3401 then?

Reply:
@mboersma yes

Reply:
note there is a separate PR in k/k to "fix" the issue but this PR mitigates the symptoms: https://github.com/kubernetes/kubernetes/pull/116879